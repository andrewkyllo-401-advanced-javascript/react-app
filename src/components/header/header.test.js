import React from 'react'
import Header from './header.js'
import { shallow } from 'enzyme'
import renderer from 'react-test-renderer'

describe('<Header />', () => {
  it('It renders correctly', () => {
    const tree = renderer.create(<Header />).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('Is alive at application start', () => {
    const component = shallow(<Header />)
    expect(component.find('header')).toExist()
  })
})