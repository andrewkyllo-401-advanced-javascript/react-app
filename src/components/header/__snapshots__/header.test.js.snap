// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> It renders correctly 1`] = `
<header>
  <h1>
    Counters R Fun
  </h1>
</header>
`;
